<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Power</class>
 <widget class="QWidget" name="Power">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>652</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>674</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:transparent;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="horizontalSpacing">
    <number>9</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="3">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="soc">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>SOC(%)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="dong_power_v">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>动力电池电压(V)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="dong_power_a">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>动力电池电流(A)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="bms">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>BMS自检状态</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_5" native="true">
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="max_power_a">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>最大可放电流(A)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="widget_6" native="true">
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="max_kw">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>最大可放电
功率(KW)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QWidget" name="widget_7" native="true">
     <property name="minimumSize">
      <size>
       <width>135</width>
       <height>135</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="max_power_v">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>最大允许电压(V)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QWidget" name="widget_8" native="true">
     <property name="minimumSize">
      <size>
       <width>135</width>
       <height>135</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/显示_参数2.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="power_state">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>电池充电状态</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>35</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="sign">
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>46</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:背景_标题2.png);
font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 28px;
color: #FFFFFF;</string>
       </property>
       <property name="text">
        <string>电池管理</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#btn_back{
background-image: url(:/按键_上一页2.png);
border:none;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
